diff --git a/dist/components/File.js b/dist/components/File.js
index 9c9da1b4dac0365f872a53fa26040039fc574fae..a12cef12fbbda878073f2a24c93d6b51b51f4f6d 100644
--- a/dist/components/File.js
+++ b/dist/components/File.js
@@ -670,7 +670,6 @@ var File = class {
 		this.headerMetadata?.remove();
 		this.headerCustom?.remove();
 		this.pre?.remove();
-		this.spriteSVG?.remove();
 		this.themeCSSStyle?.remove();
 		this.unsafeCSSStyle?.remove();
 		this.bufferAfter = void 0;
@@ -683,7 +682,6 @@ var File = class {
 		this.headerMetadata = void 0;
 		this.headerCustom = void 0;
 		this.pre = void 0;
-		this.spriteSVG = void 0;
 		this.themeCSSStyle = void 0;
 		this.appliedThemeCSS = void 0;
 		this.hasAdoptedThemeCSS = false;
diff --git a/dist/components/FileDiff.js b/dist/components/FileDiff.js
index 8d9ba31e097990b3163fd6644906144529561262..a91f7c8d128c2e37861db11f58ef0e0f9e2fd02a 100644
--- a/dist/components/FileDiff.js
+++ b/dist/components/FileDiff.js
@@ -1194,7 +1194,6 @@ var FileDiff = class {
 		this.headerMetadata?.remove();
 		this.headerCustom?.remove();
 		this.pre?.remove();
-		this.spriteSVG?.remove();
 		this.themeCSSStyle?.remove();
 		this.unsafeCSSStyle?.remove();
 		this.bufferAfter = void 0;
@@ -1209,7 +1208,6 @@ var FileDiff = class {
 		this.headerMetadata = void 0;
 		this.headerCustom = void 0;
 		this.pre = void 0;
-		this.spriteSVG = void 0;
 		this.themeCSSStyle = void 0;
 		this.appliedThemeCSS = void 0;
 		this.hasAdoptedThemeCSS = false;
diff --git a/dist/components/VirtualizedFile.js b/dist/components/VirtualizedFile.js
index 90d8a5e57f66cbe38f6b806285ecde5747bf2e16..c5fa29648b38ed447e74e837485f6d8266b241f1 100644
--- a/dist/components/VirtualizedFile.js
+++ b/dist/components/VirtualizedFile.js
@@ -447,7 +447,7 @@ var VirtualizedFile = class extends File {
 		return this.isAdvancedMode() || super.shouldDisableVirtualizationBuffers();
 	}
 	shouldGuardRebuildScroll() {
-		return false;
+		return this.isSimpleMode() && super.shouldGuardRebuildScroll();
 	}
 	isSimpleMode() {
 		return this.virtualizer.type === "simple";
diff --git a/dist/utils/platform.js b/dist/utils/platform.js
index 745b9fca5395223bc19871b8a0638acb940ded25..ce5bbdbd9f7683bed133f465971e66147b7c04b8 100644
--- a/dist/utils/platform.js
+++ b/dist/utils/platform.js
@@ -20,7 +20,7 @@ function isLinux() {
 	return _isLinux ??= /Linux/i.test(getPlatform());
 }
 function isSafari() {
-	return _isSafari ??= "safari" in window && "pushNotification" in window.safari || /^((?!chrome|android).)*safari/i.test(navigator.userAgent);
+	return _isSafari ??= "safari" in window && "pushNotification" in window.safari || /^((?!chrome|android).)*safari/i.test(navigator.userAgent) || /AppleWebKit/.test(navigator.userAgent) && !/chrome|android/i.test(navigator.userAgent);
 }
 function getPlatform() {
 	const navigator = globalThis.navigator;
